src/terms/ewald.jl

```diff
@@ -25,23 +25,25 @@ function ene_ops(term::TermEwald, ψ, occ; kwargs...)
 end
 
 function forces(term::TermEwald, ψ, occ; kwargs...)
-    T = eltype(term.basis)
-    atoms = term.basis.model.atoms
-    # TODO this could be precomputed
-    # Compute forces in the "flat" representation used by ewald
-    forces_ewald = zeros(Vec3{T}, sum(length(positions) for (elem, positions) in atoms))
-    energy_ewald(term.basis.model; forces=forces_ewald)
-    # translate to the "folded" representation
-    f = [zeros(Vec3{T}, length(positions)) for (type, positions) in atoms]
-    count = 0
-    for i = 1:length(atoms)
-        for j = 1:length(atoms[i][2])
-            count += 1
-            f[i][j] += forces_ewald[count]
+    @timeit timer "ewald" begin
```

Comment:
ewald forces, or forces/ewald?

Reply:
We should really find a way of annotating the function directly and avoid these huge blocks

Reply:
Same idea as above could apply here as well, just have a `forces_ewald` function and then ` forces(bla) = forces_ewald(bla)` in one line